packages/streamline/src/components/buttons/button/button.tsx

```diff
@@ -22,8 +23,9 @@ export const Button = ({
   onPress,
   text,
   isSkeleton = false,
+  isFloating = false,
```

Comment:
Can we apply the shadow to the container of ButtonDock instead? 
This way we can avoid having shadow without ButtonDock.

Reply:
On Figma the shadow is on the buttons not on the container, not sure we will have the same result with the shadow on the container.

Reply:
I checked and we would have the expected render on iOS but not on Android.. 
Then it's not a big point, let's keep the isFloating API in Button 👍